components/filter/filter.js

```diff
@@ -397,6 +409,20 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		this._dispatchChangeEvent(dimension, { valueKey: valueKey, selected: selected });
 	}
 
+	_handleClear() {
+		const dimension = !this._activeDimensionKey ? this._dimensions[0] : this._dimensions.find(dimension => dimension.key === this._activeDimensionKey);
+
+		this._performDimensionClear(dimension);
+		this._dispatchChangeEventNow();
+		this.requestUpdate();
+
+		if (!this._activeDimensionKey) {
+			announce(this.localize('components.filter.clearAnnounceSingle'));
```

Comment:
This announces that the filters are in the process of clearing. Is there going to be some way to know when they are done clearing? Does more need to be done after _performDimensionClear is called? Just making sure I understand the flow.

Reply:
Yeah I talked to Jeff about this - ultimately it's up to the consumer to announce when the actual data on the page is done loading/updating/clearing.  We don't have a way to tell when this happens and it's probably specific to the page.  But I think it'd be a good idea to this call out - I'll add an accessibility section to remind the consumer of this!  Good point!